commit d2dff63a0f4d4eacafa16c561b4b9e32c59f248c
Author: Nathan Scott <nathans@redhat.com>
Date:   Fri Apr 8 08:19:32 2022 +1000

    build: fix translation of PCP_SYSCONFIG_DIR in pmie unit file
    
    Resolves Red Hat BZ #2072971

diff --git a/src/pmie/GNUmakefile b/src/pmie/GNUmakefile
index e0525d49e8..dc182e444a 100644
--- a/src/pmie/GNUmakefile
+++ b/src/pmie/GNUmakefile
@@ -80,6 +80,7 @@ pmie.service : pmie.service.in
 	$(SED) <$< >$@ \
 	    -e 's;@PCP_RC_DIR@;'$(PCP_RC_DIR)';' \
 	    -e 's;@PCP_RUN_DIR@;'$(PCP_RUN_DIR)';' \
+	    -e 's;@PCP_SYSCONFIG_DIR@;'$(PCP_SYSCONFIG_DIR)';' \
 	# END
 
 pmie_farm.service : pmie_farm.service.in
